language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/InvocationNodeContextProvider.java

```diff
@@ -82,8 +84,7 @@ public void sort(BallerinaCompletionContext context, T node, List<LSCompletionIt
             super.sort(context, node, completionItems);
             return;
         }
-        for (
-                LSCompletionItem completionItem : completionItems) {
+        for (LSCompletionItem completionItem : completionItems) {
```

Comment:
With this, shall we refactor lines 72-80 as well? The logic can be combined

Reply:
Done. Can you check if that was what you expected as well?

Reply:
Yes, this is what I had in mind (y)